### ISSUE TITLE
Dynamically changing preview label position

### ISSUE THREAD

**Issue by jdujava** [junegunn/fzf#3862]
### Checklist

- [X] I have read through the manual page (`man fzf`)
- [X] I have searched through the existing issues
- [ ] For bug reports, I have checked if the bug is reproducible in the latest version of fzf

### Output of `fzf --version`

0.53.0 (c4a9ccd6)

### OS

- [X] Linux
- [ ] macOS
- [ ] Windows
- [ ] Etc.

### Shell

- [ ] bash
- [X] zsh
- [ ] fish

### Problem / Steps to reproduce

Is there a way to dynamically change the preview label position? Reading through the documentation I was unable to find something along the lines of `change-preview-label-pos` action.

Since `change-preview-label` already is implemented, wouldn't it be more convenient to merge options `preview-label` and `preview-label-pos` into one, of the signature along the lines
```
--preview-label=[N:top|bottom:]LABEL
```

**Comment by junegunn** [comment]
No. But why would you do that? I can't think of a case where moving the label would benefit the user. Wouldn't it just be confusing?

**Comment by jdujava** [comment]
I am dynamically changing the preview, and I wanted to somehow differentiate between two possible modes. I was considering multiple options:
- changing the label (isn't that noticable)
- changing the preview border style
- changing the preview background (together with different border style the alternate preview could "pop-up" more)

I also considered that the alternate preview could have different position of label (top vs bottom, left vs center vs right). I wanted to try it out, and thus this feature request.

**Comment by junegunn** [comment]
So it's not something you absolutely need.

Please note that preview/border labels can show ANSI colors, it might help in your case.

```sh
fzf --preview 'cat {}' --preview-label $' \x1b[33myellow ' --bind $'space:change-preview-label: \x1b[34mblue 
```

**Comment by jdujava** [comment]
> So it's not something you absolutely need.

Certainly not. Nevertheless, it could be nice to have the option (if someone finds the free time to implement it :).

> Please note that preview/border labels can show ANSI colors, it might help in your case.

Thanks for the suggestion!

**Comment by junegunn** [comment]
Going to close this as I don't expect much demand for the feature.
